UnityProject/Assets/Scripts/Managers/AdminCommandsManager.cs

```diff
@@ -217,6 +217,9 @@ public void CmdStartRound(NetworkConnectionToClient sender = null)
 
 			if (GameManager.Instance.CurrentRoundState == RoundState.PreRound && GameManager.Instance.waitForStart)
 			{
+				if (SubsystemBehaviourQueueInit.InitializedAll == false) return;
```

Comment:
perhaps display a message in chat so they know why it does nothing

Reply:
ok